lib/pangea/constants/model_keys.dart

```diff
@@ -102,6 +102,7 @@ class ModelKey {
   static const String feedbackLang = "feedback_lang";
 
   static const String transcription = "transcription";
+  static const String botTranscription = 'botTranscription';
```

Comment:
@WilsonLe we've (mostly) got a convention of using snake_case for json keys that we should employ here.